### ISSUE TITLE
Add CMS functionality to site

### ISSUE THREAD

**Issue by struan** [mysociety/ainow#80]
Possibly using wagtail.io as that's already Django and it means we can fairly easily use the existing design.

There are two approaches I can see. One is to add wagtail in to the existing site, the other to wrap this up as a Django app and then add it to a fresh wagtail site.

We need to get a better idea of the content of the Resources and Research pages in order to create the appropriate models for them. I think the people is ok...

After that the steps are something like:

* Create apps for: People, Resources and Research
* In each of those we should have and Index page type and a page type
* They should also create the top level page in the admin to add the sub pages to
* Create About and contact pages based on a standard page (I think)
* Update design to handle this
* Update design of event page to fit in

This does mean that the admin for the events page will remain separate from the rest of the site but I think that's probably ok for expediency. It might be that we need an event model in wagtail as well to handle sponsored events as oppose to the AI Now workshop.

We probably also need to do some of #73 for this too.

**Comment by pezholio** [comment]
Right, so I think we've got a handle on this. We've got our heads around Wagtail, and we're gonna get a basic site up and running this week. We need to:

- [x] Make blocks reusable (@pezholio)
- [x] Make a People list block to display a list of people - could add a link to a more detailed bio. This can be replicated any number of times for different 'types' of people (i.e. founders, advisory council etc) (@pezholio)
- [x] Sort out styling of person block (@zarino)
- [x] Sort out styling of the homepage feature block (@zarino)
- [x] Links in a Feature need to be styled as buttons (@zarino)
- [x] Add block to pull in YouTube video (@pezholio) (This comes for free - see http://docs.wagtail.io/en/v1.9/topics/streamfield.html#embedblock)
- [x] Add listblock for list of links with heading, subheading, description and optional thumbnail (i.e. Talks section on https://artificialintelligencenow.com/)  (@pezholio)
   - [x] `blocks.FeaturedLinkBlock.subtitle` and `blocks.FeaturedLinkBlock.description` should be optional (@pezholio)
- [x] Fix events so events with schedules redirect to existing Django app pages  (@pezholio)
- [x] Make it so event pages are a streamfield with all the other blocks, and schedule dropdown  (@pezholio)
- [x] `blocks.FeatureBlock.links` should allow links to any URL, rather than just existing wagtail pages (@pezholio)
- [x] Add (optional) link field to `blocks.PersonBlock` ? (@pezholio)
- [ ] Sort out vertical rhythm of pages other than the homepage (@zarino)
